components/prism-diff.js

```diff
@@ -40,10 +40,17 @@
 		}
 
 		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
 			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
+			alias: alias,
+			inside: {
+				'line': {
+					pattern: /(.)(?=[\s\S]).*(?:\r\n?|\n)?/,
+					lookbehind: true
+				},
+				'prefix': /[\s\S]/
```

Comment:
One last thing:
Diff highlight also adds an alias according the prefix type. E.g. `<span class="token prefix inserted">+</span>`

The code for extracting the category is on line 36.

Reply:
I've added `alias` to `'prefix'`.